tlslite/constants.py

```diff
@@ -252,6 +254,20 @@ class CertificateStatusType(TLSEnum):
     ocsp = 1
 
 
+class HeartbeatExtensionModes(TLSEnum):
+    """Types of heartbeat modes"""
+
+    peer_allowed_to_send_mode = 1
+    peer_not_allowed_to_send_mode = 2
```

Comment:
generally, the names used for the constants are consistent with the names defined in the RFC, why it's not the case here?